dictionaries/CallMap_81_delta.php

```diff
@@ -627,6 +627,10 @@
       'old' => ['bool', 'ldap'=>'resource', 'callback'=>'string'],
       'new' => ['bool', 'ldap'=>'LDAP\Connection', 'callback'=>'string'],
     ],
+    'mysqli::connect' => [
+      'old' => ['null|false', 'hostname='=>'string', 'username='=>'string', 'password='=>'string', 'database='=>'string', 'port='=>'int', 'socket='=>'string'],
+      'new' => ['bool', 'hostname='=>'string', 'username='=>'string', 'password='=>'string', 'database='=>'string', 'port='=>'int', 'socket='=>'string'],
```

Comment:
Great find. TIL about `php --rf` argument. I think its time to read `man php` properly. Thanks for the knowledge.
I've updated mysqli::__construct too, since they are almost same.
I guess I should submit a PR to PHP-doc repo now to reflect this change.